Packs/CiscoWSA/Integrations/CiscoWSAV2/CiscoWSAV2.py

```diff
@@ -0,0 +1,1526 @@
+                    ftp_max_object_size_mb=ftp_max_object_size_mb,
+                )
+
+                data = {
+                    "access_policies": [
+                        {"policy_name": policy_name, "objects": objects}
+                    ]
+                }
+            else:
+                raise DemistoException("Update failed, objects were not found.")
+        else:
+            raise DemistoException("Policy was not found.")
+
+        return self._http_request(
+            "PUT",
+            f"{V3_PREFIX}/web_security/access_policies",
+            json_data=data,
+            resp_type="response",
+        )
+
+    def access_policy_anti_malware_update_request(
+        self,
+        policy_name: str,
+        web_reputation_status: Optional[str] = None,
+        file_reputation_filtering_status: Optional[str] = None,
+        file_reputation_action: Optional[str] = None,
+        anti_malware_scanning_status: Optional[str] = None,
+        suspect_user_agent_scanning: Optional[str] = None,
+        block_malware_categories: Optional[List[str]] = None,
+        block_other_categories: Optional[List[str]] = None,
+    ):
+        """
+        Update access policy's applications settings.
+
+        Args:
+            policy_name (str): Policy name to update.
+            web_reputation_status (Optional[str], optional): Web reputation status. Defaults to None.
+            file_reputation_filtering_status (Optional[str], optional): File reputation filtering status. Defaults to None.
+            file_reputation_action (Optional[str], optional): Filr reputation action. Defaults to None.
+            anti_malware_scanning_status (Optional[str], optional): Anti-malware scanning status. Defaults to None.
+            suspect_user_agent_scanning (Optional[str], optional): Suspect uset agent scanning. Defaults to None.
+            block_malware_categories (Optional[List[str]], optional): Malware categories to block. Defaults to None.
+            block_other_categories (Optional[List[str]], optional): Other categories to block. Defaults to None.
+
+        Returns:
+            Response: API response from Cisco WSA.
+        """
+        data = remove_empty_elements(
+            {
+                "access_policies": [
+                    {
+                        "policy_name": policy_name,
+                        "amw_reputation": {
+                            "web_reputation": {"filtering": web_reputation_status},
+                            "adv_malware_protection": {
+                                "file_reputation_filtering": file_reputation_filtering_status,
+                                "file_reputation": {
+                                    file_reputation_action: [
+                                        "Known Malicious and High-Risk Files"
+                                    ]
+                                    if file_reputation_action
```

Comment:
```suggestion
```

Reply:
If `file_reputation_action` will be None `remove_empty_elements` will remove it anyway.